playwright.config.ts

```diff
@@ -11,7 +11,7 @@ export default defineConfig({
   retries: process.env.CI ? 2 : 0,
   workers: process.env.CI ? 2 : undefined,
   reporter: process.env.CI ? "blob" : "html",
-  timeout: 60000,
+  timeout: process.env.CI ? 60000 : 10000,
```

Comment:
**issue**: perhaps some variable that set timeout, instead of hardcoding it?(then you can set the right timeout in the workflow, and have different locally)